entry/src/main/js/default/pages/index/index.js

```diff
@@ -0,0 +1,62 @@
+export default {
+    data: {
+        //Left
+
+        title: "This is  title",
+        leftindicator:"/common/images/header/left_indicator_on.png",
+        lowbeam: "/common/images/header/low_beam.png",
+        bluetooth:"/common/images/header/bluetooth.png",
+        sidestand:"/common/images/header/side_stand.png",
+        highbeam:"/common/images/header/high_beam_on.png",
+        indicatoroff:"/common/images/header/right_indicator_off.png",
+        fuel:"/common/images/left/fuel.png",
+        level10:"/common/images/left/level_10.png",
+        level20:"/common/images/left/level_20.png",
+        level40:"/common/images/left/level_40.png",
+        level60:"/common/images/left/level_60.png",
+        level80:"/common/images/left/level_80.png",
+        level100:"/common/images/left/level_100.png",
+        callericon:"/common/images/middle/user_pic.png",
+        callaccept:"/common/images/middle/call_connect.png",
+        callreject:"/common/images/middle/call_disconnect.png",
+        music_cover:"/common/images/right/music_image.png",
+        previousplay:"/common/images/right/previous.png",
+        pause:"/common/images/right/pause.png",
+        nextplay:"/common/images/right/next_on.png",
+        menu:"/common/images/bottom/menu.png",
+        location:"/common/images/bottom/location.png",
+        home:"/common/images/bottom/home.png",
+        headset:"/common/images/bottom/Hearphone_select.png",
+        phone:"/common/images/bottom/contact_icon.png",
+
+
+        Front30PSI:"Front 30 PSI",
+        Rear32PSI:"Rear 32 PSI",
+        E:"E",
+        R:"R",
+        SPORT:"SPORT",
+        80:"80 %",
+        110:"110",
+        Ekm:"Est Km Range Left",
+        time:"01 : 20 PM",
+        callername:"Kate Cross",
+        Incomingcall:"Incoming call ... ",
+        48:"48",
+        Kmh:"60 Km/h",
+        avg:"Average Speed",
+        Kms:"2762 Kms",
+        song_name:"In the Name of Love",
+        album_name:"Album Name",
+        music_start_timestamp:"00:00",
+        music_stop_timestamp:"04:55"
```

Comment:
Try to use the media apis to display the actual song details getting played, if no song played as of now, get the UI from designer to display accordingly.